src/routes/devRoute.js

```diff
@@ -21,6 +24,9 @@ if (env !== 'production') {
   router.post('/cancel-upvote-post', auth.dev, cancelUpvotePost);
   router.post('/downvote-post', auth.dev, downvotePost);
   router.post('/cancel-downvote-post', auth.dev, cancelDownvotePost);
+  router.post('/trigger-daily-scoring', auth.dev, triggerDailyScoring);
+  router.post('/update-user-score-phase1', auth.dev, updateUserScorePhase1);
+  router.post('/update-user-score-phase2', auth.dev, updateUserScorePhase2);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new routes `/trigger-daily-scoring`, `/update-user-score-phase1`, and `/update-user-score-phase2` are protected by `auth.dev` middleware which is good for security. However, it's important to ensure that these routes are not exposed in production environment as they seem to be developer-specific and could potentially manipulate user scores or trigger processes prematurely. If these routes are intended to be used in a production environment, consider adding additional checks or safeguards to prevent misuse.


<!-- This is an auto-generated comment by OSS CodeRabbit -->